week-8/Homework/mandatory/3-dog-photo-gallery/dog-photo.js

```diff
@@ -0,0 +1,35 @@
+let myButton = document.querySelector("#button");
+myButton.addEventListener("click",getFunctionFetch);
+function getFunctionFetch(){
+    fetch("https://dog.ceo/api/breeds/image/random")
+        .then(checkResponse)
+        .then(response => 
+            response.json()
+        )   
+        .then(data => {
+           const myImageContainer  = document.querySelector("#imageContainer")
+           let  divContainer = document.querySelector(".card-body")
+           const  myLi = document.createElement("li")
+           const myImage = document.createElement("img")
+            myImage.className += "card-img-top w-60 m-auto p-auto"
```

Comment:
The `+=` is interesting here - what it does is add this string to the existing `className` string - because you've just created the element, we know that's empty, so this is the same as `myImage.className = ...`

There's also a handy property called `classList` we can use to add classes, so that we don't need to worry about whether there are existing classes, or spaces at the end -you can read about it [here](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList) :)